src/Stripe.net/Entities/PaymentIntents/PaymentIntentLastPaymentError.cs

```diff
@@ -23,6 +23,12 @@ public class PaymentIntentLastPaymentError : StripeEntity
         [JsonProperty("param")]
         public string Param { get; set; }
 
+        [JsonProperty("payment_intent")]
```

Comment:
Should we fix this to be a StripeError instead and release as a major version?

Reply:
I'd say yes.

Reply:
(Although maybe just bundle this with the next release for the new API version?)